topics/food.html

```diff
@@ -0,0 +1,17 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="">
+    <title>Foods</title>
+</head>
+<body>
+    <h1>Favorite Foods</h1>
+   
+    <ul>
+		<li> Chicken Curry</li>
+    </ul>
+    <a href="./index.html">Back to Topics</a> | <a href="../index.html">Back to Main</a>
+</body>
+</html>
```

Comment:
looks like you are combining the topic branch with your student branch.  ideally split these out into separate branches